gensrc/proto/lake_service.proto

```diff
@@ -118,8 +118,8 @@ message TabletStatResponse {
 // Rename file txn_{tablet_id}_{txn_id} to vtxn_{tablet_id}_{version}
 message PublishLogVersionRequest {
     repeated int64 tablet_ids = 1;
-    optional int64 txn_id = 2;
-    optional int64 version = 3;
+    repeated int64 txn_ids = 2;
```

Comment:
Question: Does changing `optional` to `repeated` break wire compatibility?  During the upgrade process, it may happen that BE is newer than FE, will there be a problem here?